code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -421,6 +421,25 @@
 	beakers += B1
 	beakers += B2
 
+/obj/item/weapon/grenade/chem_grenade/bioterrorfoam
+	name = "Bio terror foam grenade"
+	desc = "Tiger Cooperative chemical foam grenade. Causes temporary irration, blindness, confusion, autism, and mutations to carbon based life forms. Contains additional spore toxin"

```

Comment:
> autism
that was a joke to be quite honest my friend
"loss of speech" or something like that
also you're missing the full stop here